pkg/controller/kogitodataindex/resource/kafka_test.go

```diff
@@ -133,37 +133,6 @@ func TestIsKafkaServerURIResolved(t *testing.T) {
 	}
 }
 
-func Test_fromKafkaToStringMap(t *testing.T) {
```

Comment:
shouldn't we keep this test?

Reply:
We don't have this function anymore, the variables are being tested in another use case.